eo-maven-plugin/pom.xml

```diff
@@ -36,6 +36,9 @@ SOFTWARE.
   <properties>
     <runtime.version>0.1.17</runtime.version>
     <maven.version>3.3.9</maven.version>
+    <kotlin.version>1.4.32</kotlin.version>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
@nlchar no need, they are coming from the parent: https://github.com/jcabi/jcabi-parent/blob/master/pom.xml#L104-L105